docs/getting-started.md

```diff
@@ -17,7 +17,6 @@ We'll be going over the following steps:
 A few requirements before we proceed!
 
  - A running OpenShift cluster (we recommend using [`minishift`](https://docs.openshift.org/latest/minishift/getting-started/installing.html))
- - `oc` binary ([installation guide here](https://docs.openshift.org/latest/minishift/getting-started/quickstart.html#starting-minishift))
```

Comment:
don't remove this part

Reply:
why not? We don't require the binary anymore (other than login/logout which is outside of Odo) to communicate with OpenShift.

Reply:
without `oc` `odo` is useless as the user doesn't' have a way to log in to OpenShift cluster.

We still need `oc` .